tests/libstorage/storageclass.go

```diff
@@ -364,3 +364,35 @@ func isLocalPV(pv k8sv1.PersistentVolume) bool {
 func isPVAvailable(pv k8sv1.PersistentVolume) bool {
 	return pv.Spec.ClaimRef == nil
 }
+
+func GetStorageClassFromCSIDriver() string {
+	virtClient := kubevirt.Client()
+
+	csiDrivers, err := virtClient.StorageV1().CSIDrivers().List(context.Background(), metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
```

Comment:
Maybe check if `csiDrivers.Items` has a length greater than 0 and return early to avoid listing all the resources below.